### PR TITLE
Fix 44 agenda expediente

### PR DESCRIPTION
**Pull request by Juerodriguez**
## Resumen

close https://github.com/PyAr/OpenLex/issues/44
close https://github.com/Juerodriguez/OpenLex/issues/19

This PR fix the issue 44 adding an error message to the user when the file field is not loaded, also changes the display of the expiration attribute when it is None for the text "Sin vencimiento".

## Checklist

- [x] Variables, funciones y comentarios en Español
- [x] Corrector ortográfico ok
- [x] Cdigo Python cumple PEP8
- [x] Cobertura de tests con Playwright
- [x] Revisores asignados (pares & al menos 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/73370773/126038518-839bfa19-7dbb-4af2-8fda-58300ac870c0.mp4

### CODE REVIEW

File: models/db_pydoctor.py [hunk 1]
```diff
@@ -210,7 +210,8 @@ def movimiento_titulo(value, row):
         'expediente_id',
         db.expediente,
         label=T('Expediente'),
-        widget=autocomplete_expte_widget),
+        #widget=autocomplete_expte_widget
```
Comment (Juerodriguez) [inline models/db_pydoctor.py:213]:
```suggestion
        widget=autocomplete_expte_widget
```

File: models/db_pydoctor.py [hunk 2]
```diff
@@ -210,7 +210,8 @@ def movimiento_titulo(value, row):
         'expediente_id',
         db.expediente,
         label=T('Expediente'),
-        widget=autocomplete_expte_widget),
+        widget=autocomplete_expte_widget
+        ),
```
Comment (eamanu) [inline models/db_pydoctor.py:214]: No entiendo este cambio

File: views/agenda/agenda.html [hunk 3]
```diff
@@ -15,3 +15,10 @@ <h1>Agenda</h1>
 {{else:}}
         {{=grid}}
 {{pass}}
+<script>
+    $(function() {
+        $("tr>td").each(function() {
+            if ($(this).html()=="None")
+                $(this).text('Sin Vencimiento');
+        });
+    });</script>
```
Comment (eamanu) [inline views/agenda/agenda.html:24]: estaría bueno que el cierre de script esté a la misma altura del que abre

File: models/db_pydoctor.py (hunk 2 above)
Comment (Juerodriguez) [reply models/db_pydoctor.py:214]: Este cambio no deberia estar

File: views/agenda/agenda.html (hunk 3 above)
Comment (Juerodriguez) [reply views/agenda/agenda.html:24]: Sisi ahi lo agrego al cambio

**Review by eamanu — APPROVED** [review]

**Comment by eamanu** [comment]
Thanks for your contribution  @Juerodriguez

**Pull request merged**